mongodb/src/main/java/com/yahoo/ycsb/db/MongoDbClient.java

```diff
@@ -72,52 +75,41 @@ public void init() throws DBException {
 
             // initialize MongoDb driver
             Properties props = getProperties();
+
+            // Just use the standard connection format URL
+            // http://docs.mongodb.org/manual/reference/connection-string/
             String url = props.getProperty("mongodb.url",
-                    "mongodb://localhost:27017");
-            database = props.getProperty("mongodb.database", "ycsb");
-            String writeConcernType = props.getProperty("mongodb.writeConcern",
```

Comment:
I will make it backwards compatible and note that the URL supersedes the old properties.